pom.xml

```diff
@@ -45,52 +53,17 @@
                     </execution>
                 </executions>
             </plugin>
-
             <plugin>
-                <!-- Building an executable JAR -->
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
-                <version>3.0.2</version>
+                <groupId>org.pitest</groupId>
+                <artifactId>pitest-maven</artifactId>
+                <version>1.1.11</version>
                 <configuration>
-                    <archive>
-                        <manifest>
-                            <addClasspath>true</addClasspath>
-                            <classpathPrefix>lib/</classpathPrefix>
-                            <mainClass>ru.ayaz.Server</mainClass>
-                        </manifest>
-                    </archive>
+                    <excludedMethods>
+                        <param>hashCode</param>
+                        <param>equals</param>
+                    </excludedMethods>
                 </configuration>
             </plugin>
-
-            <plugin>
-                <!-- Executing JAR-->
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>exec-maven-plugin</artifactId>
```

Comment:
We don't need to create executable jar or any jar for that matter as of now.
It was necessary to run a chat server, but now we don't have it.
And SpringBoot will take care of running our app.